console.go

```diff
@@ -401,24 +401,29 @@ func consoleDefaultFormatTimestamp(timeFormat string, location *time.Location, n
 }
 
 func consoleDefaultFormatLevel(noColor bool) Formatter {
+	const unknownLevel = "???"
+	stripLevel := func(ll string) string {
```

Comment:
Any reason for this function not to be top level?

Reply:
By default, I try to reduce the scope of visibility as much as possible (especially when the stuff is too specific and doesn't look like it's to be reused elsewhere), but I don't have any strong preferences. So, if you think it's better to extract it from the function to the package, I'll do it promptly. :)

Reply:
I prefer smaller unit functions as they enhance readability and do not eliminate the possibility of inlining.